Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/overlay/InputOverlay.java

```diff
@@ -606,33 +613,66 @@ public void refreshControls()
     overlayDpads.removeAll(overlayDpads);
     overlayJoysticks.removeAll(overlayJoysticks);
 
+    String orientation =
+            getResources().getConfiguration().orientation == Configuration.ORIENTATION_PORTRAIT ?
+                    "-Portrait" : "";
+
     // Add all the enabled overlay items back to the HashSet.
     if (EmulationActivity.isGameCubeGame() || mPreferences.getInt("wiiController", 3) == 0)
     {
-      addGameCubeOverlayControls();
+      addGameCubeOverlayControls(orientation);
     }
     else if (mPreferences.getInt("wiiController", 3) == 4)
     {
-      addClassicOverlayControls();
+      addClassicOverlayControls(orientation);
     }
     else
     {
-      addWiimoteOverlayControls();
+      addWiimoteOverlayControls(orientation);
       if (mPreferences.getInt("wiiController", 3) == 3)
       {
-        addNunchukOverlayControls();
+        addNunchukOverlayControls(orientation);
       }
     }
 
     invalidate();
   }
 
-  private void saveControlPosition(int sharedPrefsId, int x, int y)
+  public void resetButtonPlacement()
+  {
+    boolean isLandscape =
+            getResources().getConfiguration().orientation == Configuration.ORIENTATION_LANDSCAPE;
+
+    if (EmulationActivity.isGameCubeGame() || mPreferences.getInt("wiiController", 3) == 0)
```

Comment:
Would you mind leaving a comment where those `wiiController` values 0 and 4 come from (or even replace them with enums/finals)?